app/models/chargeSummary/ChargeSummaryViewModel.scala

```diff
@@ -41,8 +41,12 @@ case class ChargeSummaryViewModel(
                                    origin: Option[String] = None,
                                    gatewayPage: Option[GatewayPage] = None,
                                    isMFADebit: Boolean,
+                                   isReviewAndReconcilePoaOneDebit: Boolean,
```

Comment:
we really need to create an enum something like DebitType or similar, as there can be only one debit selected for this model: Mfa, ReviewAndReconcilePoaOne or ReviewAndReconcilePoaTwo